ara/ui/templates/partials/task_index_row.html

```diff
@@ -0,0 +1,61 @@
+{% load datetime_formatting %}
+{% load truncatepath %}
+{% load static_url %}
+<tr>
+  <td class="text-center">
+    {% include "partials/task_status_icon.html" with status=task.status %}
+  </td>
+  <td class="text-center">
+    {% url 'ui:playbook' task.playbook.id as playbook_url %}
+    <a role="button" class="btn btn-outline-primary btn-link btn-sm" href="{% static_url playbook_url %}?task={{ task.id }}#results">
+      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-box-arrow-up-right" viewBox="0 0 16 16">
+        <path fill-rule="evenodd" d="M8.636 3.5a.5.5 0 0 0-.5-.5H1.5A1.5 1.5 0 0 0 0 4.5v10A1.5 1.5 0 0 0 1.5 16h10a1.5 1.5 0 0 0 1.5-1.5V7.864a.5.5 0 0 0-1 0V14.5a.5.5 0 0 1-.5.5h-10a.5.5 0 0 1-.5-.5v-10a.5.5 0 0 1 .5-.5h6.636a.5.5 0 0 0 .5-.5z"></path>
+        <path fill-rule="evenodd" d="M16 .5a.5.5 0 0 0-.5-.5h-5a.5.5 0 0 0 0 1h3.793L6.146 9.146a.5.5 0 1 0 .708.708L15 1.707V5.5a.5.5 0 0 0 1 0v-5z"></path>
+      </svg>
+    </a>
+  </td>
+  <td class="text-nowrap">{{ task.started | format_date }}</td>
+  <td class="text-nowrap">{{ task.duration | format_duration }}</td>
+  <td class="text-center">{{ task.items.results }}</td>
+  <td class="text-center">
+    <span class="badge badge-pill badge-secondary" class="btn btn-secondary" data-toggle="tooltip"  data-placement="right" data-html="true" title="{% if task.tags %}{% include 'partials/tags_list.html' with tags=task.tags %}{% endif %}">
+      {{ task.tags | length }}
+    </span>
+  </td>
+  <td>
+    <a {% if not static_generation %}href="{% url 'ui:task_index' %}?action={{ task.action }}" title="Search tasks for {{ task.action }}"{% endif %}>
+      {{ task.action }}
+    </a>
+  </td>
+  <td>
+    <a {% if not static_generation %}href="{% url 'ui:task_index' %}?name={{ task.name }}" title="Search tasks for {{ task.name }}"{% endif %}>
+      {{ task.name }}
+    </a>
+  </td>
+  <td class="text-nowrap" title="{{ task.path }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:task_index' %}?path={{ task.path }}">{{ task.path | truncatepath:25 }}</a>
```

Comment:
Same thought as playbook_path: https://github.com/ansible-community/ara/pull/342/files#r967882221, maybe we should include the task name when searching for the path so it doesn't return all the tasks in that file ?

Reply:
I thought about it and let's go with the current implementation.
We can always make the search more precise later if we change our mind.